examples/tests/trainer/test_trainer_ext.py

```diff
@@ -60,44 +63,46 @@ def require_apex(test_case):
         return test_case
 
 
-class TestFinetuneTrainer(TestCasePlus):
-    def finetune_trainer_quick(self, distributed=None, extra_args_str=None):
+class TestTrainerExt(TestCasePlus):
```

Comment:
I don't get what `Ext` means? Is it for extended? I don't really care for the class name, but the filename should have the full name so `test_trainer_extended.py`.
It's also actually testing the `Seq2SeqTrainer` in practice.

Reply:
Well, just wanted to differentiate it from test_trainer. I think it'll go through a refinement later to probably separate seq2seq distributed tests from integration tests, as they are in one pile at the moment. This is just because they reuse the same helper methods. It will evolve and improve as we expand more tests.

I don't think the filename has to match class name - this is because we have multiple cases where the filename contains multiple class names.

Bottom line, this is a WIP, but I had to change the original `Finetune`... Totally your call, @sgugger, on how do you prefer I name it.

Reply:
I don't mind if the filename doesn't match the class, it's just that `test_trainer_ext.py` is not very informative and I have no idea what's tested inside just by looking at it ;-)
Since we are in tests/trainer, the trainer part can be dropped, so there is plenty of space to be more informative. With your plans to separate, it could be `test_integration` and `test_distributed` once split.

Reply:
Yes, that's the idea for when the dust settles - there are at least 3 types of tests in there at the moment.

That is it will not be called `test_trainer_ext.py` for long, how about I called it `test_to_be_sorted.py` for now so it's loud and clear it's a pile of things.  Or can just rename it back to what it is now under `seq2seq`.

The point of this PR is to transition to the new tool to enable Suraj to move along. And I will work in future PRs to make things intuitive and named well. I will not forget.

Reply:
Sounds good to me!